### ISSUE TITLE
[6] CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features

### ISSUE THREAD

**Issue by techzzt** [techzzt/pytorch-paper-implement#6]
> **CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features**

![image](https://user-images.githubusercontent.com/49440852/178466514-9e8d7ca2-6b74-4a14-856e-e6c9941cacb2.png)

Figure 1: Class activation mapping (CAM) [52] visualizations on ‘Saint Bernard’ and ‘Miniature Poodle’ samples using various augmentation techniques

- Regularization과 robustness를 위해 mixed된 이미지 데이터 기반의 learning method 제안 
- 원본 이미지에 다른 라벨 데이터의 일정 비율로 concat한 입력 데이터를 사용하여 학습하는 알고리즘 

![image](https://user-images.githubusercontent.com/49440852/178468352-f7d65819-9cbb-486d-83b3-f870a568b866.png)

- 기존에는 원본 데이터와 동일한 크기의 다른 라벨 데이터(yB)를 합치거나 특정 구간을 마스킹하는 방법론을 사용한 반면, 제안 방법론은 다른 범주의 데이터를 일부 사용하는 방법을 사용 
- 이는 regional dropout을 적용하는 동시에 usage of full image region, mixed image & label 정보를 보완하는 것을 의미
- 본 논문에서 cropping region은 uniform 기반으로 정의되며 CAM으로 주요 영역을 시각화한 결과 분류 label과 동일한 구간에 대해서만 activation이 되는 것을 확인할 수 있음 

![image](https://user-images.githubusercontent.com/49440852/178468825-753c2358-913b-4144-acaa-6117a6a48879.png)
- label에 대한 반영 비율을 다르게 설정했을 때 ImageNet에 대해 유의미한 성능이 도출된 것을 확인할 수 있고, cutmix augmentation이 regularization 및 robustness에 효과적으로 작용했음을 확인 가능함 
- 기존 라벨과 mixed된 이미지의 라벨에 대한 공통적인 부분을 제외하고 원본 라벨의 주요 영역만을 학습함으로써 기존 모델에서 학습하지 않았던 특징 (털의 질감, 색 등)을 학습할 수 있음
